2022 - Horus Heresy.gst

```diff
@@ -266,7 +266,7 @@ During Reactions made in any Phase, a unit equipped with Jump PAcks may not acti
           <modifiers>
             <modifier type="set" field="hidden" value="true">
               <conditions>
-                <condition field="selections" scope="force" value="0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" childId="a2cf-c464-bfdd-4467" type="equalTo"/>
+                <condition type="equalTo" value="0" field="selections" scope="force" childId="a2cf-c464-bfdd-4467" shared="true" percentValue="false" includeChildSelections="true" includeChildForces="false"/>
```

Comment:
What exactly is this change?

Looks like it's just clearing up how Sagyar Mazan logic works?

Reply:
If you could that'd be great, there's a fix that needs doing for Kharash so best not to mess with it just yet

Reply:
Looking at this it does just seem to be re-ordering of the conditions. No harm no foul, happy to leave it as is